contact.html

```diff
@@ -123,12 +123,12 @@
                 <div class="nav-side">
                     <div class="d-flex align-items-center">
                         <span class="nav-item">
-                            <a class="nav-link" href="page-contact-5.html">
+                            <a class="nav-link" href="#">
                                 <i class="bi bi-person fs-5 me-2"></i>
                                 sign in
                             </a>
                         </span>

```

Comment:
Looks like this is a sign in link.

Are there any plans on building an app @engramar or @ZizhangOu? Or is this purely scoped as a static site only.

If it's a static, then we should look at removing this.

Reply:
just updated the links  to their pages,and logo link to home page.